### DISCUSSION TITLE
Geomap plugin - Adding support for dashboard variables in symbology style rules

### DISCUSSION THREAD

**Discussion by robhamnett** [grafana/grafana#53635] [Dashboards, Panels & Visualizations]
It would be very useful to be able to define style rules on static JSON geometry utilizing variables in the definitions. See Attached image for example. A use case might be a user selects a location from a variable drop down and it would highlight the point on the map, or increase it's diameter, etc. Thanks!

![05ce2360-4f82-4440-8ee1-5ac97f5611c9](https://user-images.githubusercontent.com/65688161/184253716-34b0620b-602f-4a52-82e1-2e362d992f91.jpg)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
